frontend/src/app/about/page.tsx

```
@@ -0,0 +1,103 @@
+          <div className="flex flex-col justify-center items-center gap-4 w-full max-w-[272px] md:max-w-[600px] lg:max-w-[1120px]">
+            <h1 className="text-[32px] leading-[40px] md:text-[44px] md:leading-[52px] lg:text-[57px] lg:leading-[64px] font-medium text-center text-white w-full lg:tracking-[-0.25px]">
+              Nacimos para cuidar de ti y tu cabello
+            </h1>
+            <p className="text-[28px] leading-[36px] md:text-[26px] md:leading-[34px] lg:text-[28px] lg:leading-[36px] font-normal text-center text-white w-full">
+              Descubre la historia, la pasión y las personas detrás de los productos que transformarán tu cabello.
+            </p>
+          </div>
+
+          <div className="flex flex-row items-center gap-4">
+            <button className="flex justify-center items-center h-12 hover:opacity-90 transition-opacity">
+              <div className="flex justify-center items-center px-4 py-[10px] w-full h-10 bg-[#215E6B] rounded-full">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)